it/services/EncryptionServiceTest.scala

```diff
@@ -17,17 +17,17 @@
 package services
 
 import models.mongo.EncryptedPensionsUserData
-import utils.IntegrationTest
+import utils.{IntegrationTest, PensionUserDataStub}
 
 class EncryptionServiceTest extends IntegrationTest {
 
   private val underTest: EncryptionService = app.injector.instanceOf[EncryptionService]
 
   "encryptUserData" should {
-    val data = pensionsUserData.copy()
+    val data = PensionUserDataStub.pensionUserData().copy()
```

Comment:
Coverage is quite low on the encryption service as nothing is being decrypted/encrypted
If you change as below coverage of the encryption service will increase by 20%

Can you change this 
`val data = PensionUserDataStub.pensionUserData().copy()`

to this:

```
val paymentsIntoPensionViewModel: PaymentsIntoPensionViewModel = PaymentsIntoPensionViewModel(Some(true),
    Some(222.3),Some(true), Some(22.44), Some(true), Some(44))
```

Then change PensionsUserData to use it for your default value as  below

```
def pensionUserData(
                       sessionId: String = "sessionid",
                       mtdItId: String = "1234567890",
                       nino: String = "nino",
                       taxyear: Int = taxYear,
                       isPriorSubmission: Boolean = true,
                       cya: Option[PensionsCYAModel] = Some(PensionsCYAModel(Some(paymentsIntoPensionViewModel)))
                     ): PensionsUserData = {
    PensionsUserData(
      sessionId = sessionId,
      mtdItId = mtdItId, nino = nino, taxYear = taxyear, isPriorSubmission = isPriorSubmission, pensions = cya, lastUpdated = testClock.now()
    )
```